tests/integration/test_arteria_runfolder.py

```diff
@@ -198,6 +203,14 @@ async def test_get_runfolders_path_missing_runfolder(client, config, runfolder):
         assert resp.reason == "Runfolder '200624_A00834_0183_FAKE_RUNFOLDER' does not exist"
 
 
+def test_route_param_format():
+    correct_route = "/api/1.0/runfolders/path{runfolder:/.*}"
+
+    for route in routes._items:     # pylint: disable=protected-access
+        if '{runfolder:' in route.path:
+            assert route.path == correct_route
+
+
```

Comment:
This test does not execute code from the service, i.e. it's not a real regression test, I don't think it is necessary either.

The reason this bug was not detected by our test base was because the url we used in our tests contained two `/` in a row, e.g. `/api/1.0/runfolders/path//tmp/tmpppapz5ec/200624_A00834_0183_BHMTFYDRXX`. A better test could work as follow:
1. build a url targeting a runfolder from a monitored directory
1. before sending the request, make sure the url does not contain two `/` in a row after `runfolders/path`
2. Send the request and make sure the service finds the runfolder

This could be incorporated in `test_post_runfolders_path` and `test_get_runfolder_path` for instance.